core/src/main/java/hudson/Functions.java

```diff
@@ -1574,10 +1575,6 @@ public static String toEmailSafeString(String projectName) {
         return projectName;
     }
 
-    public String getSystemProperty(String key) {
```

Comment:
Caused JENKINS-48593 which was fixed in https://github.com/jenkinsci/jenkins/pull/3200